doc/sphinx/source/utils.rst

```diff
@@ -240,9 +240,16 @@ The following parameters have to be set in the script in order to make it run:
 * ``outputs``, *str*: Name of the directory in which the job outputs (.out and .err files) are going to be saved. The outputs will be saved in `/home/user/<outputs>`.
 * ``conda_path``, *str*: Full path to the `mambaforge/etc/profile.d/conda.sh` executable.
 
-The script will generate a submission script for all recipes using by default the ``compute`` queue and with a time limit of 8h. In case a recipe
+Optionally, the following parameters can be edited:
+
+* ``env``, *str*: Path to ``config-user.yml`` if default ``~/.esmvaltool/config-user.yml`` not used.
```

Comment:
There seems to be a mismatch in here

Reply:
Well spotted! Fixed in 7fcaf836504a1cbbca1fd01b55779d0e96fdc8c8